lib/src/navbar/navbar.component.ts

```diff
@@ -18,7 +19,38 @@ export class NavbarComponent implements OnInit {
     constructor() {
     }
 
+    menuOpen: boolean = false;
+    hamburger: string = 'inline';
+    hamburgerClose: string = 'none';
+
+    @ContentChildren(NavbarMenuComponent) mobileMenu: QueryList <NavbarMenuComponent>;
+
     ngOnInit() {
     }
 
+    toggleMobileMenu() {
+        if( this.mobileMenu.first ) {
+            if( this.menuOpen ) {
+                this.mobileMenu.first.hide();
+                this.menuOpen = false;
+                this.hamburger = 'inline';
+                this.hamburgerClose = 'none';
+            }
+            else {
+                this.mobileMenu.first.show();
+                this.menuOpen = true;
+                this.hamburger = 'none';
+                this.hamburgerClose = 'inline';
+            }
+        }
+    }
+
+    menuClick( event ) {
+        let clickTarget: string = event.target.classList[0];
+
+        //Verify that the click in the mobile menu came from a navigation item
+        if( clickTarget.indexOf("menu-dropdown") == -1 )
```

Comment:
Using a `===` instead of `==` will also check the data type and is usually a good idea.